manifests/plugin/supervisory_authority.pp

```diff
@@ -0,0 +1,45 @@
+# = Foreman supervisory_authority Plugin
+#
+# This class installs the supervisory_authority plugin and configuration file
+#
+# === Parameters:
```

Comment:
Would it make sense to split this up into `Parameters` and `Advanced Parameters`? If not, I prefer the puppet-strings syntax since it's the officially supported one. However, RDoc has the feature of parameter groups which we do use in our installer.

Reply:
I added the "Advanced Parameters" section with all parameters which are optional.
The doc-block was taken from another plugin (memcache) to be consistent with whats already there.
I think we should rework the RDoc in all the Puppet files at once to not have different styles of documenting stuff.
What do you think?